civis/io/_tables.py

```diff
@@ -234,6 +234,11 @@ def read_civis_sql(sql, database, use_pandas=False, job_name=None,
                                                    db_id, credential_id,
                                                    polling_interval)
 
+    # if we retrieved headers then we are performing a parallel unload
+    # in which case we need to specify backslash as the escapechar
+    if headers:
```

Comment:
Could we do `if headers is not None:` here?  In case we ever return a numpy array or something else that might cause this to break?

Reply:
updated